src/lib/server/voice/frameBus.ts

```
@@ -0,0 +1,25 @@
+    if (!this.handlers[type]) {
+      this.handlers[type] = [];
+    }
+    this.handlers[type].push(fn as (frame: Frame) => void);
+  }
+
+  publish(frame: Frame) {
+    const subs = this.handlers[frame.type];
+    if (!subs) {
+      return;
+    }
+    subs.forEach((fn) => fn(frame));
```

Biome (2.4.10)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)